openfortivpn: add page (#4902)
tldr-pages/tldr: pages/linux/openfortivpn.md

# openfortivpn

> > Client that connects to a VPN.

Reviewer: Based on `openconnect` for Cisco:
```suggestion
> A VPN client, for Fortinet's proprietary PPP+SSL VPN solution.
```

- Connect to a VPN with a username and password:

> `openfortivpn -u {{username}} -p {{password}}`

Reviewer: We prefer the long argument format:
```suggestion
`openfortivpn --username={{username}} --password={{password}}`
```

> - Connect to a VPN defined in a {{/etc/openfortivpn/config}} file:

Reviewer: ```suggestion
- Connect to a VPN using a specific configuration file (defaults to `/etc/openfortivpn/config`):
```

> `sudo openfortivpn -c config`

Reviewer: The `--config` is visible in `--help`:
```suggestion
`sudo openfortivpn --config=path/to/config`
```

# openfortivpn

> > A VPN client, for Fortinet's proprietary PPP+SSL VPN solution.

Reviewer: Oh, I found the repo:
```suggestion
> A VPN client, for Fortinet's proprietary PPP+SSL VPN solution.
> More information: <https://github.com/adrienverge/openfortivpn>.
```

Author: Thanks for the link! I added it :+1:

# openfortivpn

> A VPN client, for Fortinet's proprietary PPP+SSL VPN solution.
> More information: <https://github.com/adrienverge/openfortivpn>.

- Connect to a VPN with a username and password:

`openfortivpn --username={{username}} --password={{password}}`

- Connect to a VPN using a specific configuration file (defaults to `/etc/openfortivpn/config`):

> `sudo openfortivpn --config=path/to/config`

Reviewer 2: Perhaps this will read better?
```suggestion
`sudo openfortivpn --config={{path/to/config}}`
```
